doc/changelog/08-vernac-commands-and-options/16911-rm-tc-filtered-unification.rst

```diff
@@ -0,0 +1,5 @@
+- **Removed:**
+  the `Typeclasses Filtered Unification` flag, deprecated
```

Comment:
maybe this should repeat that the behaviour was bugged as mentioned in the deprecation changelog